plugins/aea-cli-ipfs/aea_cli_ipfs/core.py

```diff
@@ -51,7 +51,7 @@ def ipfs(click_context: click.Context) -> None:
         ipfs_tool.daemon.start()
 
 
-@ipfs.resultcallback()
+@ipfs.result_callback()
```

Comment:
@kongzii Thanks, can you fix the failing test it a trivial fix - https://github.com/valory-xyz/open-aea/actions/runs/8828297414/job/24240199769?pr=736#step:8:17631